README.md

```diff
@@ -38,6 +38,23 @@ It can then be invoked as:
 BEAKER_setfile=centos7-64 bundle exec rake beaker
 ```
 
+known working setfiles that we currently (2022-01-06) use:
+
+* ubuntu1604-64
+* ubuntu1804-64
+* ubuntu2004-64
+* ubuntu2204-64
+* debian9-64
+* debian10-64
+* debian11-64
+* centos7-64
+* centos8-64
+* centos9-64
+* almalinux8-64
+* almalinux9-64
+* rocky8-64
+* rocky9-64
```

Comment:
I dislike static lists in README files, but wonder what would be better. In the past we had a rake task to list setfiles, but it was based on listing files. Perhaps we should implement something in puppet_metadata to print all the setfiles based on metadata.json.

Reply:
yes, I'm also not a big fan of this :(

Reply:
Implemented in https://github.com/voxpupuli/puppet_metadata/pull/142